app/controllers/root_controller.rb

```diff
@@ -109,12 +110,58 @@ def publication
       end
 
       @interaction_details = prepare_interaction_details(@publication, authority_slug, snac)
+    elsif @publication.format == 'local_transaction'
+      @location = fetch_location @postcode
+
+      if @location.nil?
+        @postcode_error = "fullPostcodeNoMapitMatch"
+      elsif @location.try(:code) == 400
+        @postcode_error = "invalidPostcodeFormat"
+      end
```

Comment:
Is a `400` the only possible code here?  What about other errors - don't we log them?